### PR TITLE
#160820759 Send Email Notification for Posted Comment

### PR DESCRIPTION
**Pull request by albert85**
#### What does this PR do?
send email notification for a posted comment

#### Description of Task to be completed?
- create an email notification engine
- send email notification for posted comment by the requester.
- send email notification for posted comment by the manager

#### How should this be manually tested?
- Clone the repo - git clone https://github.com/andela/travel_tool_back.git
- Enter into the project directory - cd travel_tool_back
- Install dependencies and start the server - make start
- Start the frontend.
- Make a request and choose yourself as the manager
- Post a comment on the request details
- Check your gmail account for a new message from travela regarding the posted comment
- Click on the view request button to redirect to the request

#### Any background context you want to provide?
- In other to test this feature, you need to update your role on the Users table to that of a manager.
Manager roleId: 50319
- Check the `env.example` to check the variable that is required to implement this task which is `SENDGRID_API_KEY`, `REACT_APP_AUTH_REDIRECT_URL` and `APP_EMAIL`

#### What are the relevant pivotal tracker stories?
[#160820759](https://www.pivotaltracker.com/story/show/160820759)

#### Screenshots (if appropriate)
screen shot 2018-10-04 at 12 00 04 pm

#### Questions:
N/A

### CODE REVIEW

File: src/helpers/email/__tests__/switchMessage.test.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+describe('switchMessage helper', () => {
+  it('should return new Request message', (done) => {
+    const receivedMessage = switchMessage(
+      { type: 'New Request', senderName: 'Tester' }
+    );
+    expect(receivedMessage.split(' ')).toContain('<b>Tester</b>');
+    expect(receivedMessage.split(' ')[8]).toEqual('<b>Tester</b>');
+    done();
+  });
+  it('should return posted commebt message', (done) => {
```
Comment (clintfidel) [inline src/helpers/email/__tests__/switchMessage.test.js]: please check for typo, it should be `comment` not `commebt`

Comment (albert85) [reply src/helpers/email/__tests__/switchMessage.test.js]: thanks @clintfidel

File: src/helpers/email/switchMessage.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      );
+    case 'Comments':
+      return (
+        `
+      <b>${msgDetail.senderName}</b> posted a comment.
+      Login to your travela account for details.
+      
+      `
+
+      );
```
Comment (amarachukwu-agbo) [inline src/helpers/email/switchMessage.js]: Please remove extra spaces

Comment (albert85) [reply src/helpers/email/switchMessage.js]: @amarachukwu-agbo done

**Comment by clintfidel** [comment]
Please resolve merge conflicts @albert85

**Comment by babadee001** [comment]
@albert85 There is a drop in coverage on this PR.

**Pull request merged**